apps/stable_diffusion/src/utils/stable_args.py

```diff
@@ -29,6 +29,12 @@ def path_expand(s):
     help="text you don't want to see in the generated image.",
 )
 
+p.add_argument(
+    "--img_path",
+    type=str,
+    help="Path to the image input for img2img/inpaint",
+)
+
```

Comment:
there should be two cli args, right? How are you taking care of inputs wrt masking?